plugins.xml

```diff
@@ -7173,6 +7173,52 @@
 			<institution>Reviewer Credits GmbH</institution>
 			<email>emre.danisan@reviewercredits.com</email>
 		</maintainer>
+		
+		<release date="2025-03-20" version="3.3.0.9" md5="4618a32978c65733f8b6e65b5de78572">
+			<package>https://www.reviewercredits.com/wp-content/uploads/reviewer-credits/ojsplugin/V3009/reviewerCredits.tar.gz</package>
+			<compatibility application="ojs2">
+				<version>3.3.0.0</version>
+				<version>3.3.0.1</version>
+				<version>3.3.0.2</version>
+				<version>3.3.0.3</version>
+				<version>3.3.0.4</version>
+				<version>3.3.0.5</version>
+				<version>3.3.0.6</version>
+				<version>3.3.0.7</version>
+				<version>3.3.0.8</version>
+				<version>3.3.0.9</version>
+				<version>3.3.0.10</version>
+				<version>3.3.0.11</version>
```

Comment:
The `<version>3.3.0.11</version>` through `<version>3.3.0.19</version>` lines are unnecessary, since they will be covered by the `<version>~3.3.0.0</version>` semver pattern. OJS 3.3.0-x prior to 3.3.0-11 didn't support this, which is why they're listed individually. The same goes for all the individual 3.4.0-x entries; you can just use `<version>~3.4.0.0</version>` in their place.